### ISSUE TITLE
There is no helm chart

### ISSUE THREAD

**Issue by madejackson** [docker-mailserver/docker-mailserver-helm#107]
There is no helm chart to install where there should be one, at least according to docs: https://github.com/docker-mailserver/docker-mailserver-helm/tree/master/charts/docker-mailserver

installing is not possible:
```
Release "docker-mailserver" does not exist. Installing it now.
Error: non-absolute URLs should be in form of repo_name/path_to_chart, got: docker-mailserver
```

**Comment by cfis** [comment]
You need to first tell Helm about the repo:

```
helm repo add docker-mailserver https://docker-mailserver.github.io/docker-mailserver-helm
```

Will need to update documentation (or feel free to create a PR and I'll apply it).

**Comment by madejackson** [comment]
Thx, fixed docs with: #108
